svc-aggregation/agmodel/system.go

```diff
@@ -316,21 +316,21 @@ func deletefilteredkeys(key string) error {
 	}
 	for _, value := range sf.SearchKeys {
 		for k := range value {
-			delErr := conn.Del(k, key)
+			delErr := conn.Del(k, 0, "*"+key+"*")
```

Comment:
Please check for all the possibilities of key deletion using *

Reply:
addressed